docs/data-cookbooks/browse-generation.md

```diff
@@ -78,173 +76,205 @@ For this example, you are going to be adding two workflows to your Cumulus deplo
-          cmrMetadataFormat: '{$.meta.cmrMetadataFormat}'
-          additionalUrls: '{$.meta.additionalUrls}'
-          generateFakeBrowse: true
-    Type: Task
-    Resource: ${FakeProcessingLambdaFunction.Arn}
-    Catch:
-      - ErrorEquals:
-        - States.ALL
-        ResultPath: '$.exception'
-        Next: WorkflowFailed
-    Retry:
-      - ErrorEquals:
-          - States.ALL
-        IntervalSeconds: 2
-        MaxAttempts: 3
-    Next: FilesToGranulesStep
+```json
+"ProcessingStep": {
+  "Parameters": {
+    "cma": {
+      "event.$": "$",
+      "task_config": {
+        "bucket": "{$.meta.buckets.internal.name}",
+        "collection": "{$.meta.collection}",
+        "cmrMetadataFormat": "{$.meta.cmrMetadataFormat}",
+        "additionalUrls": "{$.meta.additionalUrls}",
+        "generateFakeBrowse": true,
+        "cumulus_message": {
+          "outputs": [
+            {
+              "source": "{$.granules}",
+              "destination": "{$.meta.input_granules}"
+            },
+            {
+              "source": "{$.files}",
+              "destination": "{$.payload}"
+            }
+          ]
+        }
+      }
+    }
+  },
+  "Type": "Task",
+  "Resource": "${module.cumulus.fake_processing_task_lambda_function_arn}",
+  "Catch": [
+    {
+      "ErrorEquals": [
+        "States.ALL"
+      ],
+      "ResultPath": "$.exception",
+      "Next": "WorkflowFailed"
+    }
+  ],
+  "Retry": [
+    {
+      "ErrorEquals": [
+        "States.ALL"
+      ],
+      "IntervalSeconds": 2,
+      "MaxAttempts": 3
+    }
+  ],
+  "Next": "FilesToGranulesStep"
+},
 ```
 
-**Please note**: ```FakeProcessing``` is the core provided browse/cmr generation we're using for the example in this entry.
-
  If you're not ingesting mock data matching the example, or would like to use modify the example to ingest your own data please see the [build-lambda](#build-lambda) section below.    You will need to configure a different lambda entry for your lambda and utilize it in place of the ```Resource``` defined in the example workflow.
 
-#### Cumulus Configuration
+ **Please note**: ```FakeProcessing``` is the core provided browse/cmr generation lambda we're using for the example in this entry.
 
-In an editor, open app/config.yml and modify your stepFunctions key to contain the file you just created:
+#### Lambdas
 
-```yaml
-stepFunctions: !!files [
-  {some list of workflows},
-  'browseExample.yml'
-]
-```
+All lambdas utilized in this example are provided in a standard deployment of Cumulus and require no additional configuration.
 
-This will cause kes to export the workflows in the new file along with the other workflows configured for your deployment.
+If you're not ingesting mock data matching the example, or would like to use this entry to ingest your own data please see the [build-lambda](#build-lambda) section below.    You will need to add a terraform ```aws_lambda_function``` resource to your deployment.
```

Comment:
Seems like this is duplicated above?

https://github.com/nasa/cumulus/pull/1283/files#diff-231365a3a612192224b4dbc98817eae7R188

Reply:
f52fad5a0